src/control/cmd/dmg/pretty/system.go

```diff
@@ -162,9 +182,14 @@ func printSystemResults(out, outErr io.Writer, results system.MemberResults, abs
 		return nil
 	}
 
-	if err := printSystemResultTable(out, results, absentRanks); err != nil {
-		return err
+	err := printSystemResultTable(out, getRandStr, results, absentRanks)
```

Comment:
I am not very found of this try and test way for checking if the random string separator is OK.
Could it make sense to use a non printable characters as separators, and thus to not have to use a random string ?

Reply:
+1 to this